docs/how_to_publish_to_cmr_with_earthdata_varinfo.ipynb

```diff
@@ -251,6 +253,27 @@
     ")"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Instantiate a VarInfoFromNetCDF4 object with an optional config file. This will override any default configuration."
```

Comment:
```suggestion
    "Instantiate a VarInfoFromNetCDF4 object with an optional config file. This will override default configuration."
```
I don't think having a config fill will override Any defaults.  This just sounds better to me.